bin/seed-database.ts

```diff
@@ -50,13 +96,42 @@ export default async function seedDatabase(): Promise<void> {
       };
     });
   try {
-    await Promise.all(mockUsers.map(prisma.user.create));
+    await Promise.all(mockPlayers.map(prisma.user.create));
+    usersCreateMessage.text = "Created 10 players.";
+    usersCreateMessage.succeed();
   } catch (err) {
-    usersCreateMessage.fail(`Creating users failed\n\n${err.message}`);
-    process.exit(1);
+    usersCreateMessage.fail(`Creating players failed\n\n${err.message}`);
   }
-  usersCreateMessage.text = "Created 10 players.";
-  usersCreateMessage.succeed();
+
+  const mentorsCreateMessage = Ora(`Creating ${NUMBER_USERS} mentors`).start();
+  const mockMentors: UserCreateArgs[] = Array(NUMBER_USERS)
```

Comment:
was curious why id's isn't a property included in the viewee data since it's a required field for ViewingPermissionCreateWithoutViewerInput